en/docs/learn/examples/rest-api-examples/publishing-a-swagger-api.md

```diff
@@ -38,10 +36,8 @@ Following is a sample REST API configuration with a custom Swagger definition. S
                 <args/>
             </payloadFactory>
             <loopback/>
```

Comment:
Let's remove loopback, as now we have removed outsequence from the config, this does not have any meaning.